BaseFilter/sections/foreign.txt

```diff
@@ -6121,28 +6116,13 @@ tu.no##.col-full-width > .products-carousel.daily-offers
 ! https://github.com/AdguardTeam/AdguardFilters/issues/21961
 ||nakenprat.com/adfirst*.js
 ||nakenprat.com/adlast*.php^
-! https://github.com/AdguardTeam/AdguardFilters/issues/20808
-itavisen.no##.banner
-! https://github.com/AdguardTeam/AdguardFilters/issues/16687
-finn.no##iframe#easyad
-finn.no##iframe#smallEasyAd
-finn.no##.banner-skyscraper-container
-finn.no##div.banners[style^="height: 150px"]
-finn.no##.banners[data-banner-pos="topbanner"]
-finn.no##div.external-banner-board.adsbox[data-controller]
-finn.no##.banners.adsbox
```

Comment:
Hm. I don't see finn.no rules in Finnish or your filters.

Reply:
The site received a massive complete overhaul a year or 2 ago, so older entries are simply not needed in any way.

I have `finn.no##.banners` in Nordic Filters at least, from what I can see.

Reply:
Further clarification: Of the 7 entries that are commented on, only `finn.no##.banners.adsbox` still matches anything at all, and even then only on the frontpage. The other 6 entries aren't matching any elements at all.